### PR TITLE
disable unsuitable controls for hierCluster

### PR DESCRIPTION
**Pull request by congyu-lu**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/matrix.controls.js [hunk 1]
```diff
@@ -399,7 +399,10 @@ export class MatrixControls {
 								width: 50,
 								align: 'center',
 								chartType: 'matrix',
-								inputs: [{ label: 'N/A' }, { settingsKey: 'rowh', min: 8, max: 30, step: 1 }]
+								inputs: [{ label: 'N/A' }, { settingsKey: 'rowh', min: 8, max: 30, step: 1 }],
+								getDisplayStyle(plot) {
```
Comment (siosonel) [inline client/plots/matrix.controls.js:403]: @congyu-lu User's may prefer a short row height to be able to view more rows without having to scroll.

File: client/plots/matrix.controls.js [hunk 2]
```diff
@@ -429,7 +432,10 @@ export class MatrixControls {
 								inputs: [
 									{ settingsKey: 'colspace', min: 0, max: 20, step: 1 },
 									{ settingsKey: 'rowspace', min: 0, max: 20, step: 1 }
-								]
+								],
```
Comment (siosonel) [inline client/plots/matrix.controls.js:435]: @congyu-lu Users may prefer wider gap between heatmap columns, especially in small cohorts. If a heatmap should not have column spacing, then the default value should be 0.

File: client/plots/matrix.controls.js (hunk 1 above)
Comment (congyu-lu) [reply client/plots/matrix.controls.js:403]: changing row height now will make the left dendrogram misaligned. should we disable it for now before it's fixed?

File: client/plots/matrix.controls.js (hunk 2 above)
Comment (congyu-lu) [reply client/plots/matrix.controls.js:435]: similar to rowh changing these will make the dendrograms misaligned. should we disable it for now before it's fixed?

**Review by xzhou82 — APPROVED** [review]

File: client/plots/matrix.controls.js (hunk 1 above)
Comment (siosonel) [reply client/plots/matrix.controls.js:403]: @congyu-lu we should try to fix this in a new branch.

**Pull request merged**